lib/galaxy/tools/__init__.py

```diff
@@ -1860,8 +1860,8 @@ def populate_model( inputs, state_inputs, group_inputs, other_values=None ):
 
         # create tool model
         tool_model = self.to_dict( request_context )
-        tool_model[ 'inputs' ] = []
-        populate_model( self.inputs, state_inputs, tool_model[ 'inputs' ] )
+        tool_model[ 'inputs' ] = {}
+        self.populate_model( request_context, self.inputs, state_inputs, tool_model[ 'inputs' ] )
```

Comment:
Changes to this file look like a bad merge to me, and it breaks the workflow editor.